UnityProject/Assets/Scripts/Items/Implants/Organs/Tongue.cs

```diff
@@ -12,6 +12,10 @@ public class Tongue : BodyPartFunctionality
 		[SerializeField]
 		private List<LanguageSO> languages = new List<LanguageSO>();
 
+		private bool cannotSpeak = false;
```

Comment:
No need to split it in a field and a property.
```suggestion
		public bool CannotSpeak {get; private set}
```

Reply:
I also have a concern with the name. Normally using negation to name bools can be confusing.

Reply:
well, mix logics and plus I don't have to do ! before passing it to functions